packages/playground/src/hooks/useOnline.ts

```diff
@@ -0,0 +1,32 @@
+import { computed, onMounted, onUnmounted, ref } from "vue";
+
+export function useOnline() {
+  const online = ref(true);
+
+  function setOnline(value: boolean) {
+    return () => {
+      online.value = value;
+    };
+  }
+
+  const isOnline = setOnline(true);
```

Comment:
why do we need 2 variables and listeners? doesn't a false isOnline indicate that it is in fact offline?

Reply:
variables for listeners to we need a ref to callback to remove it in removeEventListener

Reply:
Yes but why isOffline necessary? Why not just use isOnline because when isOnline=false that is offline

Reply:
it's just setting the value nothing more